addons/survey/views/survey_report_templates.xml

```diff
@@ -15,7 +15,15 @@
                     <div class="certification-content">
                         <div t-if="user_input.scoring_success">
                             <p>This certificate is presented to
-                                <br/><span class="user-name" t-esc="user_input.partner_id.name or user_input.email"/>
+                                <br/>
+                                <t t-set="certif_style" t-value=""/>
+                                <t t-if="user_input.partner_id.name.isupper()">
+                                    <t t-set="certif_style" t-value="certif_style + 'font-family: certification-serif'"/>
```

Comment:
`';'` missing

If you are in upper and with len > 20 you will have:
`'font-family: certification-seriffont-size: 40px; line-height:4;'`

Reply:
Indeed! Thanks for the correction**s**